test/e2e/servicebindingrequest_test.go

```diff
@@ -186,6 +187,7 @@ func assertDeploymentEnvFrom(
 // assertSBRStatus will determine if SBR is on "success" state
 func assertSBRStatus(
 	ctx context.Context,
+	t *testing.T,
```

Comment:
Test functions should always receive `t` as first argument.

```suggestion
	t *testing.T,
	ctx context.Context,
```

Reply:
Please disregard this comment; `ctx` should always be the first argument per https://github.com/golang/go/wiki/CodeReviewComments#contexts.